learn/pkg/learn/frontmatter.go

```diff
@@ -9,12 +9,14 @@ type (
 	frontmatterType string
 	answerType      string
 	difficulty      string
+	verification    string
 )
 
 var (
 	validFrontmatterTypes = []string{"course", "unit", "unittest", "quiz", "exercise", "question"}
 	validAnswerTypes      = []string{"single-choice", "multiple-choice", "free-text", "multiple-free-texts", "program"}
 	validDifficulties     = []string{"easy", "medium", "hard", "retriable"}
+	validVerifications    = []string{"match" /* default */, "none", "parse-error", "no-parse-error"}
```

Comment:
I had a little trouble grokking at first the difference between `match` and `no-parse-error` since a `match` answer should not have parse errors. But now I realise the two parse-error forms are for "pick the program that compiles" and "pick the program that doesn't compile". I think. I'm not sure if there's a place to note that, as there's not really any docs for the frontmatter in more detail. Perhaps that can come later.

:pineapple: